tools/ci_build/github/azure-pipelines/nuget/templates/windowsai.yml

```diff
@@ -576,31 +576,31 @@ jobs:
         msbuild /p:Platform=x64 Microsoft.AI.MachineLearning.Tests.vcxproj
       workingDirectory: $(Build.SourcesDirectory)\csharp\test\Microsoft.AI.MachineLearning.Tests
 
-  - task: PowerShell@2	
```

Comment:
This file contains a lot of tabs, which should be avoided. So I changed them to spaces.